<?xml version="1.0" ?>
<sdf xmlns:xacro="http://www.ros.org/wiki/xacro" version="1.8">
    <world name="bot_world">
    
        <!--SETS UP PHYSICS PLUGINS, GROUND, AND SUN-->
        <xacro:include filename="world.xacro"/>


        <!--SETS UP THE STEWART PLATFORM-->
        <xacro:include filename="robot.xacro"/>

    </world>
</sdf>